src/lib/wagmi.ts

```diff
@@ -85,6 +89,13 @@ export const WAGMI_SUPPORTED_CHAINS: Chain[] = [
   kava,
 ];
 
+const selectedNetwork = getLocalStorage({
```

Comment:
nit: maybe rename these `initialSelectedNetwork` and `initialChainId` so it's clear to them that these are just the init values